MFVNeutralino/plugins/Vertexer.cc

```diff
@@ -860,11 +1107,22 @@ void MFVVertexer::produce(edm::Event& event, const edm::EventSetup& setup) {
         reco::Vertex vnm1(TransientVertex(kv_reco->vertex(ttks)));
         const double dist3_2 = mag2(vnm1.x() - v[0]->x(), vnm1.y() - v[0]->y(), vnm1.z() - v[0]->z());
         const double distz = mag(vnm1.z() - v[0]->z());
-        if (verbose) printf("  refit %lu chi2 %7.4f vtx %7.4f %7.4f %7.4f dist3 %7.4f distz %7.4f\n", i, vnm1.chi2(), vnm1.x(), vnm1.y(), vnm1.z(), sqrt(dist3_2), distz);
+        //const double distz_error = sqrt(vnm1.covariance(2, 2) + v[0]->covariance(2, 2));
+        //const double distz_sig = distz/distz_error;
+        //if (histos){
+        //  h_noshare_nm1refit_dz->Fill(distz);
+        //  h_noshare_nm1refit_dz_err->Fill(distz_error);
+        //  h_noshare_nm1refit_dz_sig->Fill(distz_sig);
+        //  h_noshare_nm1refit_dz_dzerr->Fill(distz, distz_error);
+        //}
```

Comment:
I think for the vertexer (here and elsewhere in the file) we should remove things that we aren't actually using, only because the vertexer is complicated!